tasks/static_quality_gates/experimental_gates.py

```diff
@@ -75,41 +159,59 @@ class InPlaceArtifactReport:
     file_inventory: list[FileInfo]
 
     # Metadata
+    # TODO(agent-build): integrate GateMetricHandler metadata
     measurement_timestamp: str
     pipeline_id: str
     commit_sha: str
     arch: str
     os: str
     build_job_name: str
 
+    # Docker-specific metadata (optional)
+    docker_info: DockerImageInfo | None = None
+
     def __post_init__(self):
         """Validate report data"""
         if not self.artifact_path:
             raise ValueError("artifact_path cannot be empty")
         if not self.gate_name:
             raise ValueError("gate_name cannot be empty")
         if self.on_wire_size < 0:
-            raise ValueError("on_wire_size must be non-negative")
+            raise ValueError("on_wire_size must be positive")
         if self.on_disk_size < 0:
-            raise ValueError("on_disk_size must be non-negative")
+            raise ValueError("on_disk_size must be positive")
```

Comment:
To be confirmed by a native English speaker but I think that, in English,
* `positive` means `> 0` whereas
* `non-negative` means `⩾ 0`.

So, technically, `non-negative` was more correct in this error message since `0` isn’t `positive` but doesn’t raise this error.